vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,324 @@
+
+Once we import the dataset correctly with Formatter, we need to pre-process the data which is taken care by dataProcess.
+
+This step is essentially Data pre-processing and quality control of MS runs of the original raw data into quantitative data for model fitting and group comparison.
+
+There are 3 main steps that happen in background :
+
+* __Normalization__ - There are three different normalizations supported. 'equalizeMedians' (default) represents constant normalization (equalizing the medians) based on reference signals is performed. 'quantile' represents quantile normalization based on reference signals is performed. 'globalStandards' represents normalization with global standards proteins. FALSE represents no normalization is performed.
+
+* __Feature selection__ - This also has three options i.e. Select All features, Top-N features (by mean intensity) or “Best” features.
+* __Missing value imputation__ - We impute plausible values in case of missing data points. The RunLevelData can be queried to show Number of imputed intensities (censored intensities) in a RUN and Protein.
+
+
+``` {r code dataProcess}
+summarized = dataProcess(pd_imported)
+
+head(summarized$FeatureLevelData)
+
+head(summarized$ProteinLevelData)
+
+head(summarized$SummaryMethod)
+
+```
+
+### __1.4.1 Data Process Plots__
+
+After dataProcessing the input data, MSstats provides multiple plots to analyze the pre-processed data. Here we show the various types of plots we can use. Once invoked, a pdf file will be downloaded with corresponding feature level data and the Plot generated.
+
+```{r dataProcessPlots, results='hide', message=FALSE, warning=FALSE}
+
+# Profile plot
+dataProcessPlots(data=summarized, type="ProfilePlot", address = FALSE, which.Protein = "P0ABU9")
+
+# Quality control plot
+dataProcessPlots(data=summarized, type="QCPlot", address = FALSE, which.Protein = "P0ABU9")
+
+# Quantification plot for conditions
+dataProcessPlots(data=summarized, type="ConditionPlot", address = FALSE, which.Protein = "P0ABU9")
+
+```
+
+
+
+
+### __1.5 Modeling __
+
+In this step we test for significant changes in protein abundance across conditions based on a family of linear mixed-effects models in targeted Selected Reaction Monitoring (SRM), Data-Dependent Acquisition (DDA or shotgun), and Data-Independent Acquisition (DIA or SWATH-MS) experiment. 
+
+It is applicable to multiple types of sample preparation, including label-free workflows, workflows that use stable isotope labeled reference proteins and peptides, and workflows that use fractionation. Experimental design of case-control study (patients are not repeatedly measured) or time course study (patients are repeatedly measured) is automatically determined based on proper statistical model.
+
+
+``` {r code groupComparison}
+
+# Model
+labels = unique(summarized$ProteinLevelData$GROUP)
+contrast_matrix = MSstatsContrastMatrix('pairwise', labels)
+
+model = groupComparison(contrast_matrix, summarized)
```

Comment:
I think you can get rid of lines 211-212 and just do `model = groupComparison("pairwise", summarized)`